suite/runsuite.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Bash is cool, but for a Python project I think it's better to use `unittest` from the standard library. It will be a lot easier to maintain as this project matures, and it has more features geared to testing.